<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>tgMainWindowClass</class>
 <widget class="QMainWindow" name="tgMainWindowClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>801</width>
    <height>608</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>tgMainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget"/>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>801</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="actionAdd_Source"/>
    <addaction name="actionAdd_Target"/>
   </widget>
   <widget class="QMenu" name="menuRender">
    <property name="title">
     <string>Render</string>
    </property>
    <widget class="QMenu" name="menuObject">
     <property name="title">
      <string>Object</string>
     </property>
     <addaction name="actionShow_Model"/>
     <addaction name="actionShow_ConvexHull"/>
    </widget>
    <widget class="QMenu" name="menuMode">
     <property name="title">
      <string>Mode</string>
     </property>
     <addaction name="actionRender_Points"/>
     <addaction name="actionRender_Wire"/>
     <addaction name="actionRender_FlatWire"/>
     <addaction name="actionRender_Flat"/>
    </widget>
    <addaction name="menuObject"/>
    <addaction name="menuMode"/>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menuRender"/>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <property name="windowTitle">
    <string>Toolbar</string>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="actionAdd_Source"/>
   <addaction name="actionAdd_Target"/>
   <addaction name="separator"/>
   <addaction name="actionShow_Model"/>
   <addaction name="actionShow_ConvexHull"/>
   <addaction name="separator"/>
   <addaction name="actionRender_Points"/>
   <addaction name="actionRender_Wire"/>
   <addaction name="actionRender_FlatWire"/>
   <addaction name="actionRender_Flat"/>
  </widget>
  <widget class="QStatusBar" name="statusBar">
   <property name="sizeGripEnabled">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QDockWidget" name="dockWidgetConsole">
   <property name="windowTitle">
    <string>Console</string>
   </property>
   <attribute name="dockWidgetArea">
    <number>8</number>
   </attribute>
   <widget class="QWidget" name="dockWidgetContents"/>
  </widget>
  <action name="actionAdd_Source">
   <property name="icon">
    <iconset resource="tgMainWindow.qrc">
     <normaloff>:/tgMainWindow/Resources/add.png</normaloff>:/tgMainWindow/Resources/add.png</iconset>
   </property>
   <property name="text">
    <string>Add Source</string>
   </property>
   <property name="toolTip">
    <string>Add Source</string>
   </property>
  </action>
  <action name="actionAdd_Target">
   <property name="icon">
    <iconset resource="tgMainWindow.qrc">
     <normaloff>:/tgMainWindow/Resources/add.png</normaloff>:/tgMainWindow/Resources/add.png</iconset>
   </property>
   <property name="text">
    <string>Add Target</string>
   </property>
   <property name="toolTip">
    <string>Add Target</string>
   </property>
  </action>
  <action name="actionRender_Points">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Points</string>
   </property>
   <property name="toolTip">
    <string>Render Points</string>
   </property>
  </action>
  <action name="actionRender_Flat">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Flat</string>
   </property>
  </action>
  <action name="actionRender_Wire">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Wire</string>
   </property>
   <property name="toolTip">
    <string>Render Wire</string>
   </property>
  </action>
  <action name="actionRender_FlatWire">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Flat Wire</string>
   </property>
   <property name="toolTip">
    <string>Render Flat Wire</string>
   </property>
  </action>
  <action name="actionShow_Model">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Model</string>
   </property>
  </action>
  <action name="actionShow_ConvexHull">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Convex Hull</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="tgMainWindow.qrc"/>
 </resources>
 <connections/>
</ui>
